src/libtelio.udl

```diff
@@ -694,7 +694,7 @@ dictionary FeatureFirewall {
     /// Turns on connection resets upon VPN server change (Deprecated alias for neptun_reset_conns)
     boolean boringtun_reset_conns;
     /// Custom private IP range
-    Ipv4Net? custom_private_ip_range;
+    Ipv4Net? exclude_private_ip_range;
```

Comment:
It would be nice if the doc comment for this field was a bit more descriptive

Reply:
Done ✅